EstateReporting.IntegrationTests/Shared/SharedSteps.cs

```diff
@@ -702,8 +702,94 @@ public async Task GivenIMakeTheFollowingManualMerchantDeposits(Table table)
             }
         }
 
-        [When(@"I get the Transactions By Report for '(.*)' with the Start Date ""(.*)"" and the End Date ""(.*)"" the following data is returned")]
-        public async Task WhenIGetTheTransactionsByReportForWithTheStartDateAndTheEndDateTheFollowingDataIsReturned(String estateName, String startDateString, String endDateString, Table table)
+        //[When(@"I get the Transactions By Report for Estate '(.*)' with the Start Date ""(.*)"" and the End Date ""(.*)"" the following data is returned")]
+        //public async Task WhenIGetTheTransactionsByReportForEstateWithTheStartDateAndTheEndDateTheFollowingDataIsReturned(String estateName, String startDateString, String endDateString, Table table)
+        //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6340917)